src/ui/buttons/styles/responsive.js

```diff
@@ -147,15 +150,34 @@ export function buttonResponsiveStyle({
       }] .${CLASS.BUTTON_LABEL} .${CLASS.SPACE} {
                     line-height: ${perc(buttonHeight, 50) + 5}px;
                 }
+                
+                .${CLASS.BUTTON}.${CLASS.BORDER_RADIUS} {
+                  ${borderRadius >= 0 ? `border-radius: ${borderRadius}px` : ""}
+                }
+
+                .${CLASS.BUTTON}.${CLASS.SHAPE}-${BUTTON_SHAPE.SHARP} {
+                  border-radius: 0px;
+                }
 
                 .${CLASS.BUTTON}.${CLASS.SHAPE}-${BUTTON_SHAPE.RECT} {
                     border-radius: 4px;
                 }
 
                 .${CLASS.BUTTON}.${CLASS.SHAPE}-${BUTTON_SHAPE.PILL} {
-                    border-radius: ${Math.ceil(buttonHeight / 2)}px;
+                    border-radius: ${pillBorderRadius}px;
+                }
+
+                .${CLASS.BUTTON_ROW}.${CLASS.BORDER_RADIUS} .menu-button {
+                  ${borderRadius >= 0 ? `border-radius: ${borderRadius}px` : ""}
                 }
 
+                .${CLASS.BUTTON_ROW}.${CLASS.SHAPE}-${
+        BUTTON_SHAPE.SHARP
+      } .menu-button {
```

Comment:
Could we add some images of the menu button with the various border radius please?

Reply:
The menu button is technically out of scope for our phase 1 deliverables, so we might get push back from design about holding off on these changes.

Reply:
So glad you told me to add this because it was definitely broken. Here's the new version: 
![image](https://github.com/paypal/paypal-checkout-components/assets/60204385/d91783cf-f7a8-4715-8a4b-370f1c3bebb5)
![image](https://github.com/paypal/paypal-checkout-components/assets/60204385/434d4740-9d86-41fb-86ea-b6de46c3d1ad)
![Uploading image.png…]()